pkgs/development/python-modules/import-expression/default.nix

```diff
@@ -6,26 +6,29 @@
   pytestCheckHook,
   astunparse,
   setuptools,
+  typing-extensions,
 }:
 buildPythonPackage rec {
   pname = "import-expression";
-  version = "1.1.5";
+  version = "2.0.0";
   pyproject = true;
 
   src = fetchPypi {
     inherit version;
     pname = "import_expression";
-    hash = "sha256-mVlYj8/I3LFEoHJRds/vbCjH2x/C1oNiUCXmh1FtQME=";
+    hash = "sha256-Biw7dIOPKbDcqYJSCyeqC/seREcVihSZuaKNFfgjTew=";
   };
 
   build-system = [ setuptools ];
-  dependencies = [ astunparse ];
+  dependencies = [
+    astunparse
```

Comment:
https://github.com/ioistired/import-expression/blob/e1b5c76912898dc6d34af7057283d0c9dcc226c0/setup.py#L127-L129

This dependency is no longer needed 
```suggestion

```